client/src/pages/Game.tsx

```diff
@@ -1,14 +1,42 @@
 import React, { FC } from 'react'
 import { RouteComponentProps } from '@reach/router'
+import { useEffect, useState } from 'react'
+import * as SignalR from '@microsoft/signalr'
+import Button from '../components/Button'
 
 export interface GameProps extends RouteComponentProps {
   id?: string
 }
 
-const Game: FC<GameProps> = ({ id }) => (
-  <>
-    This is the game with id <em>{id}</em>.
-  </>
-)
+const Game: FC<GameProps> = (props: GameProps) => {
+  const [hubConnection, setHubConnection] = useState()
+  const [counter, setCounter] = useState(0)
+
+  useEffect(() => {
+    const connection = new SignalR.HubConnectionBuilder()
+      .withUrl('https://localhost:5001/gameHub')
```

Comment:
Configure the `proxy` property in `client/package.json` and use the same origin as the client app.
```suggestion
      .withUrl('/gameHub')
```
That way we wont need to worry about CORS bulshittery

Reply:
I'll create a ticket for the proxy config.